unitTests/ui/QuickviewTest.ts

```diff
@@ -39,6 +43,32 @@ export function QuickviewTest() {
       return template;
     };
 
+    it('should render button correctly', done => {
+      expect($$(quickview.element).hasClass('coveo-accessible-button')).toBe(true);
+      expect($$(quickview.element).find('.coveo-icon-for-quickview')).not.toBeNull();
+      expect($$(quickview.element).find('.coveo-caption-for-icon')).not.toBeNull();
+      done();
+    });
+
+    it('should have a tooltip when layout is list', done => {
```

Comment:
I feel there is a slight disconnect where the test description does not reflect the test implementation, hmm. What about:
`when the layout is list, the caption should be positioned (i.e. appear as a tooltip)`
and
`when the layout is card, the caption should not be positioned (i.e. be inline)`